test/e2e/service_to_service_test.go

```diff
@@ -92,7 +92,7 @@ func sendRequest(t *testing.T, clients *test.Clients, resolvableDomain bool, dom
 	return client.Do(req)
 }
 
-func testProxyToHelloworld(t *testing.T, clients *test.Clients, helloworldDomain string, inject bool) {
+func testProxyToHelloworld(t *testing.T, clients *test.Clients, helloworldDomain string, inject bool, accessibleExternal bool) {
```

Comment:
changes under "test/e2e/" directory should belong to the separate PR for the E2E test, right?